06-eventing/knative/event-source-broker.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: sources.eventing.knative.dev/v1alpha1
+kind: ContainerSource
+metadata:
+  name: heartbeat-event-source
+spec:
+  image: docker.io/matzew/kube-heartbeat
```

Comment:
This image needs to be pushed to RH Developers Quay account and referenced like that.
cc @kameshsampath 

This is the source repo: https://github.com/matzew/heartbeat

Reply:
@aliok how about 

`quay.io/openshift-knative/knative-eventing-sources-heartbeats:v0.5.0`   ?

Reply:
that's basically our image for

https://github.com/knative/eventing-sources/blob/v0.5.0/cmd/heartbeats/main.go

Reply:
Thanks @matzew. Adapted the YAML and the docs.